### PR TITLE
fix getPointsForMessage method

### PR DESCRIPTION
**Pull request by josephmturner**
(it now returns reference points for each log in the reference history of all reference points contained in the message)

### CODE REVIEW

**Comment by josephmturner** [comment]
This commit is necessary for this ushin-ui-components [PR](https://github.com/USHIN-Inc/ushin-ui-components/pull/154).

File: index.js [hunk 1]
```diff
@@ -171,24 +171,22 @@ class USHINBase {
   }
 
   async getPointsForMessage({ main, shapes }) {
-    const allPoints = new Set();
+    const pointIds = [main, ...Object.values(shapes).flat()];
```
Comment (RangerMauve) [inline index.js:175]: Maybe we should just use the `allPoinnts` array from the message instead of `main` and `shapes`? :P

**Review by RangerMauve — CHANGES_REQUESTED** [review]
This looks good, I think we might be able to get rid of most of the logic if we use `message.allPoints` instead of calculating all the points here.

**Comment by josephmturner** [comment]
Works great! We can delete the extra comment in a later commit :)

**Pull request merged**